log::debug and ignore on daemon usage in CI (#9237)
vercel/turborepo: docs/repo-docs/reference/configuration.mdx

}
```

<Callout type="good-to-know">

>   Note that when running in a CI environment the daemon is disabled, even if you
>   set this value to `true`. This is because the daemon is not useful in a CI
>   environment and can cause timing out on deploys.

Reviewer: ```suggestion
  When running in a CI environment the daemon is always disabled regardless of this setting.
```
